### ISSUE TITLE
2.1.2-linux-x86_64.AppImage - CONTRACTS - blank window

### ISSUE THREAD

**Issue by SvenMeyer** [trufflesuite/ganache#1616]
OS : Linux Manjaro
Truffle v5.1.13-nodeLTS.0 (core: 5.1.12)
Solidity v0.5.16 (solc-js)
Node v12.14.1
Web3.js v1.2.1

ganache-2.1.2-linux-x86_64.AppImage

- created anew workspace with 'Quickstart'
- added truffle-config.js to projects
- [save and restart]
- deployed contract with truffle (`truffle migrate --network gg  --reset`)

ganache GUI Interface : 
CONTRACTS
`Migrations`as well as my contract `ProofOfAsset`show a [DEPLOYED] icon
(Why a TX count of 33 ?)

![ganache-2 1 2-linux-x86_64 AppImage_CONTRACTS_Screenshot](https://user-images.githubusercontent.com/25609/74078980-28923e00-4a85-11ea-8f61-9648690f0d55.png)

When I click on my contract , for a second some text is being displayed then just a blank white windows.

Inspect - console : 

```
/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:187 TypeError: Cannot read property 'to' of null
    at content.props.transactions.map.tx (/tmp/.mount_ganachJi3IAS/resources/app.asar/src/renderer/screens/transactions/TxList.js:49:17)
    at Array.map (native)
    at TxList.render (/tmp/.mount_ganachJi3IAS/resources/app.asar/src/renderer/screens/transactions/TxList.js:47:43)
    at Mh (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:173:192)
    at Lh (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:172:282)
    at mj (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:247:361)
    at jj (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:216:164)
    at Z (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:210:471)
    at $i (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:206:372)
    at /tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:114:115
    at exports.unstable_runWithPriority (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/scheduler/cjs/scheduler.production.min.js:20:134)
    at vf (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:113:325)
    at yf (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:114:61)
    at O (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:113:476)
    at eg (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:204:235)
    at Object.enqueueSetState (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:127:237)
di @ /tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:187
/tmp/.mount_ganachJi3IAS/resources/app.asar/src/renderer/screens/transactions/TxList.js:49 Uncaught (in promise) TypeError: Cannot read property 'to' of null
    at content.props.transactions.map.tx (/tmp/.mount_ganachJi3IAS/resources/app.asar/src/renderer/screens/transactions/TxList.js:49:17)
    at Array.map (native)
    at TxList.render (/tmp/.mount_ganachJi3IAS/resources/app.asar/src/renderer/screens/transactions/TxList.js:47:43)
    at Mh (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:173:192)
    at Lh (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:172:282)
    at mj (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:247:361)
    at jj (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:216:164)
    at Z (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:210:471)
    at $i (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:206:372)
    at /tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:114:115
    at exports.unstable_runWithPriority (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/scheduler/cjs/scheduler.production.min.js:20:134)
    at vf (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:113:325)
    at yf (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:114:61)
    at O (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:113:476)
    at eg (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:204:235)
    at Object.enqueueSetState (/tmp/.mount_ganachJi3IAS/resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:127:237)
```

**Comment by SvenMeyer** [comment]
looks similar to https://github.com/trufflesuite/ganache/issues/1315 from June 2019

**Comment by davidmurdoch** [comment]
Fixed by aad867c3d2f53009b653b701efe349ef0c1b0925
